sources/web/datalab/polymer/modules/ApiManager.ts

```diff
@@ -143,6 +144,18 @@ class ApiManager {
   static xsrfToken = '';
 
   /**
+   * Current connection status. Set to the last connection status.
+   */
+  static isConnected = true;
+
+  /**
+   * Handlers for connected/disconnected status. These will be called when the
+   * isConnected property changes value.
+   */
+  static connectedHandler: Function;
```

Comment:
What is the method signature for these? () => void?

Seems like it's common enough that they should be proper events.